lib/core/test/unit/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobinSpec.hs

```diff
@@ -1576,6 +1838,194 @@ unit_makeChangeForUserSpecifiedAsset =
     assetC :: AssetId
     assetC = AssetId (UnsafeTokenPolicyId $ Hash "A") (UnsafeTokenName "2")
 
+--------------------------------------------------------------------------------
+-- Equipartitioning natural numbers
+--------------------------------------------------------------------------------
+
+-- Test that natural numbers are equipartitioned fairly:
+--
+-- Each portion must be within unity of the ideal portion.
+--
+prop_equipartitionNatural_fair
+    :: Natural -> NonEmpty () -> Property
+prop_equipartitionNatural_fair n count =
+    prop_equipartitionNatural_fair_inner $ equipartitionNatural n count
+
+prop_equipartitionNatural_fair_inner :: NonEmpty Natural -> Property
```

Comment:
This function can actually now be inlined into `prop_equipartitionNatural_fair`.